eva-accession-release-automation/run_release_in_embassy/add_mapping_weight_attribute.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
```

Comment:
Sorry, this was initially designed to be part of the workflow but since we did this as a one-time update, it shouldn't belong here any longer... Will move this...